table_processing/table_tools.py

```diff
@@ -40,3 +40,44 @@ def remove_duplicate_limits(limit_list, threshold):
 
 def within_threshold(a, b, threshold):
     return abs(b - a) < abs(threshold)
+
+
+def _calculate_raw_row_column_limits(bbox_list):
+    # Use boundaries of bounding boxes to set row and column limits
+    row_limits = []
+    column_limits = []
+    for box in bbox_list:
+        x1,y1,x2,y2 = box
+        row_limits.append(y1)
+        row_limits.append(y2)
+        column_limits.append(x1)
+        column_limits.append(x2)
+    row_limits.sort()
+    column_limits.sort()
+
+    return(column_limits,row_limits)
+    
+
+def _calculate_row_column_limits(image_dim, bbox_list):
+    column_limits, row_limits =_calculate_raw_row_column_limits(bbox_list)
+    threshold = 2  # no row or column can be only 2 pixels wide
+    row_limits = remove_duplicate_limits(row_limits, threshold)
+    column_limits = remove_duplicate_limits(column_limits, threshold)
+
```

Comment:
we should look to tune this once merged to reflect the changes made to the  _calculate_row_column_limits() function in Table.py from the previous recent PRs

Reply:
Ya this is gonna have to be tweaked to reflect the changes you guys made in the last pr once merged

Reply:
I suggest a rebase from main and rerun to confirm that things are looking good.